types/matrix-js-sdk/index.d.ts

```diff
@@ -683,12 +684,12 @@ export class Filter {
 }
 
 export class MatrixEvent {
-  event: any;       //  The raw (possibly encrypted) event. Do not access this property directly unless you absolutely have to. Prefer the getter methods defined
-  sender: RoomMember;   //  The room member who sent this event, or null e.g. this is a presence event. This is only guaranteed to be set for events that appear in
-  target: RoomMember;   //  The room member who is the target of this event, e.g. the invitee, the person being banned, etc.
-  status: EventStatus;  //  The sending status of the event.
-  error: Error;        //  most recent error associated with sending the event, if any
-  forwardLooking: boolean;      //  True if this event is 'forward looking', meaning that getDirectionalContent() will return event.content and not event.prev_content.
+  event: RawEvent;         //  The raw (possibly encrypted) event. Do not access this property directly unless you absolutely have to. Prefer the getter methods defined
+  sender: RoomMember;      //  The room member who sent this event, or null e.g. this is a presence event. This is only guaranteed to be set for events that appear in
+  target: RoomMember;      //  The room member who is the target of this event, e.g. the invitee, the person being banned, etc.
+  status: EventStatus;     //  The sending status of the event.
+  error: Error;            //  most recent error associated with sending the event, if any
+  forwardLooking: boolean; //  True if this event is 'forward looking', meaning that getDirectionalContent() will return event.content and not event.prev_content.
 
   constructor(event: object)
   getType(): EventType;
```

Comment:
Can you please also add the `getContent` getter, since you're at it?

Reply:
wow, that was quick, sure, I'll add the `getType` and `getContent` :)

Reply:
done, along with some other getters based on https://github.com/matrix-org/matrix-js-sdk/blob/fb8976167190e81e4139da8501068c3bd0523d82/src/models/event.js#L185-L345